src/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/Util/CompilationSettings.cs

```diff
@@ -56,8 +59,17 @@ internal static class CompilationSettingsHelper {
                 ttl = DefaultCompilerServerTTLInDevEnvironment;
             }
 
-            _csc = new CompilerSettings(CompilerFullPath(@"bin\roslyn\csc.exe"), ttl);
-            _vb = new CompilerSettings(CompilerFullPath(@"bin\roslyn\vbc.exe"), ttl);
+            var customRoslynCompilerLocation = Environment.GetEnvironmentVariable(CustomRoslynCompilerLocation, EnvironmentVariableTarget.Process);
+            if(customRoslynCompilerLocation != null)
+            {
+                _csc = new CompilerSettings($"{customRoslynCompilerLocation}\\csc.exe", ttl);
```

Comment:
Is it safe to assume this C# 6 syntax is ok everywhere? I assume it gets compiled away and all. Just asking.